skins/moono/dialog.css

```
@@ -399,6 +399,11 @@ div.cke_disabled .cke_dialog_ui_labeled_content div *
 	cursor: default;
 }
 
+/* (#3433) */
+.cke_dialog_ui_labeled_required{
+	color: red;
+}
```

Let's stick to the file styling.
```suggestion
/* (#3433) */
.cke_dialog_ui_labeled_required
{
	color: red;
}
```